src/EnergyPlus/PlantUtilities.cc

```diff
@@ -2467,68 +2295,521 @@ namespace PlantUtilities {
 		//  in cases where demand side (air loop, for example) equipment is "fighting" with the plant loop
 		// The result of this routine can help the plant "lock-in" and take action to stop the iteration
 
-		// REFERENCES:
-		// na
-
 		// Using/Aliasing
 		using namespace DataPlant;
 		using namespace DataLoopNode;
 
-		// Return value
-		bool Converged;
-
-		// Locals
-		// FUNCTION ARGUMENT DEFINITIONS:
-
-		// FUNCTION PARAMETER DEFINITIONS:
-		// na
-
 		// FUNCTION LOCAL VARIABLE DECLARATIONS:
 		Real64 InletAvgTemp;
 		Real64 InletAvgMdot;
 		Real64 OutletAvgTemp;
 		Real64 OutletAvgMdot;
 
 		if ( FirstHVACIteration ) {
-			Converged = false;
-			return Converged;
+			return false;
 		}
 
 		InletAvgTemp = sum( PlantLoop( ThisLoopNum ).LoopSide( ThisLoopSide ).InletNode.TemperatureHistory ) / size( PlantLoop( ThisLoopNum ).LoopSide( ThisLoopSide ).InletNode.TemperatureHistory );
-
 		if ( any_ne( PlantLoop( ThisLoopNum ).LoopSide( ThisLoopSide ).InletNode.TemperatureHistory, InletAvgTemp ) ) {
-			Converged = false;
-			return Converged;
+			return false;
 		}
 
 		InletAvgMdot = sum( PlantLoop( ThisLoopNum ).LoopSide( ThisLoopSide ).InletNode.MassFlowRateHistory ) / size( PlantLoop( ThisLoopNum ).LoopSide( ThisLoopSide ).InletNode.MassFlowRateHistory );
-
 		if ( any_ne( PlantLoop( ThisLoopNum ).LoopSide( ThisLoopSide ).InletNode.MassFlowRateHistory, InletAvgMdot ) ) {
-			Converged = false;
-			return Converged;
+			return false;
 		}
 
 		OutletAvgTemp = sum( PlantLoop( ThisLoopNum ).LoopSide( ThisLoopSide ).OutletNode.TemperatureHistory ) / size( PlantLoop( ThisLoopNum ).LoopSide( ThisLoopSide ).OutletNode.TemperatureHistory );
-
 		if ( any_ne( PlantLoop( ThisLoopNum ).LoopSide( ThisLoopSide ).OutletNode.TemperatureHistory, OutletAvgTemp ) ) {
-			Converged = false;
-			return Converged;
+			return false;
 		}
 
 		OutletAvgMdot = sum( PlantLoop( ThisLoopNum ).LoopSide( ThisLoopSide ).OutletNode.MassFlowRateHistory ) / size( PlantLoop( ThisLoopNum ).LoopSide( ThisLoopSide ).OutletNode.MassFlowRateHistory );
-
 		if ( any_ne( PlantLoop( ThisLoopNum ).LoopSide( ThisLoopSide ).OutletNode.MassFlowRateHistory, OutletAvgMdot ) ) {
-			Converged = false;
-			return Converged;
+			return false;
 		}
 
-		//If we made it this far, we're good!
-		Converged = true;
+		// If we made it this far, we're good!
+		return true;
+
+	}
+
+
+	void
+	ScanPlantLoopsForObject(
+			std::string const & CompName,
+			int const CompType,
+			int & LoopNum,
```

Comment:
Extra tabs here, some of those happening in some of the new header files, too. Not a big deal.